tensorboard/webapp/core/actions/core_actions.ts

```diff
@@ -73,3 +75,13 @@ export const changePageSize = createAction(
   '[Core] Page Size Change',
   props<{size: number}>()
 );
+
+export const polymerInteropRunSelectionChanged = createAction(
```

Comment:
Can you explain to me why this action is prefixed with polymerInterop but the other is not?  Does it have something to do with who you expect to dispatch the action?

Reply:
I think the answer here is "we expect polymerInteropRunSelectionChanged to be dispatched due to interaction with tf-run-selector" while "we expect fetchRunSuceeded to be fired by a fetch of the runs data in angular code".  Is that correct?  Whether I'm right or wrong, it could be worth documenting who you expect to be firing the events and when you expect them to do it.

Reply:
Added a stanza about how the `polymerInteropRunSelectionChanged` is expected to only be used by tf-runs-selector.